projects/wave-core-new/src/lib/operators/dialogs/operator-list/operator-list.component.ts

```diff
@@ -128,11 +128,14 @@ export class OperatorListComponent implements OnInit {
             },
             description: 'Get statistics for raster layer'
         },
-        //     {
-        //         component: TimePlotComponent,
-        //         type: TimePlotType,
-        //         description: 'Plot time data'
-        //     }
+        {
+            component: TemporalRasterMeanPlotDialogComponent,
+            type: {
+                NAME: 'Temporal Raster Mean Plot',
+                ICON_URL: createIconDataUrl('Temporal Raster Mean Plot'),
+            },
+            description: 'Create an area chart over means of raster time series'
```

Comment:
"Create an area chart over the mean pixel values of the images of a raster time series"

Reply:
3328cae